bootstrap.sh

```diff
@@ -512,26 +512,29 @@ else
 	if hash 2>/dev/null pacman; then
 		archLinux "$emulator"
 	fi
-	# Debian or any derivative of it
-	if hash 2>/dev/null apt-get; then
-		ubuntu "$emulator" "$defpackman"
-	fi
-	# Fedora
-	if hash 2>/dev/null dnf; then
-		fedora "$emulator"
-	fi
 	# Suse and derivatives
 	if hash 2>/dev/null zypper; then
```

Comment:
Wouldn't it be more correct to convert all ifs into elifs? The fact that it's written in such way that few may be invoked at the same time smells as for me.

Reply:
Yes, that's an excellent point, let me update the PR.